terraform/aws/operator.tf

```diff
@@ -42,14 +46,7 @@ resource "helm_release" "flink_operator" {
   # Enable prometheus metrics for all
   set {
     name = "defaultConfiguration.flink-conf\\.yaml"
-    value = yamlencode({
-      "kubernetes.operator.metrics.reporter.prom.class" : "org.apache.flink.metrics.prometheus.PrometheusReporter",
-      "kubernetes.operator.metrics.reporter.prom.port" : "9999",
-      "kubernetes.jobmanager.annotations" : {
-        "prometheus.io/scrape" : "true"
-        "prometheus.io/port" : "9999"
-      }
-    })
+    value = local_file.flink_operator_config.content
```

Comment:
Can this simply be `templatefile("flink_operator_config.tpl",{ mount_path=var.historyserver_mount_path })` instead? That way, we can get rid of having to gitignore `.yaml` files and save an additional resource here. Also keeps it simpler with one fewer level of redirection.

Reply:
Actually, why is this a template at all? Can't it be just yamlencode still, with the values for `jobmanager.archive.fs.dir` be set to `var.historyserver_mount_path`? I think that's much cleaner, and we'll never run into YAML indentation issues due to how templating works.

Reply:
Could get that solution to render via `templatefile` or `yamlencode` correctly so [ticked](https://github.com/pangeo-forge/pangeo-forge-cloud-federation/issues/7) and moving onto better problems for now. Will have a look at it when I get around to getting Prometheus to work better